lib/rules/selector-max-pseudo-class/__tests__/index.js

```diff
@@ -0,0 +1,130 @@
+"use strict";
+
+const messages = require("..").messages;
+const ruleName = require("..").ruleName;
+const rules = require("../../../rules");
+
+const rule = rules[ruleName];
+
+testRule(rule, {
+  ruleName,
+  config: [0],
+  skipBasicChecks: true,
+
+  accept: [
+    {
+      code: "",
+      description: "empty stylesheet"
+    },
+    {
+      code: "a {}",
+      description: "empty rule"
+    },
+    {
+      code: '@import "foo.css";',
+      description: "blockless statement"
+    },
+    {
+      code: "a { color: pink; }",
+      description: "selector with no pseudo class"
+    },
+    {
+      code: ".foo { .bar { .baz {} } }",
+      description: "nested selectors with no pseudo classes"
+    }
+  ],
+
+  reject: [
```

Comment:
Can we add `line` and `column` properties and values to all the `reject` tests, please? e.g.

```
{
  code: ".foo { .bar { .baz {} } }",
  description: "nested selectors with no pseudo classes"
  line: 1,
  column: 1
}
```